ibm_mq/tests/test_ibm_mq_unit.py

```diff
@@ -172,3 +172,26 @@ def test_channel_queue_config_error(instance_config):
         IBMMQConfig(instance_config)
 
     assert 'channel, queue_manager are required configurations' in str(excinfo.value)
+
+
+def test_ssl_connection_creation(instance):
+    """
+    Test that we are not getting unicode/bytes type error.
+
+    TODO: We should test SSL in e2e
```

Comment:
Should we log a ticket for this (if not done already)?

Reply:
Created.